BitbucketPS.psd1

```diff
@@ -0,0 +1,131 @@
+#

```

Comment:
Don't create the file again.
Just save it as `UTF-8` instead of `UTF-16 BE`

Reply:
Back to original